apps/src/music/views/HeaderButtons.tsx

```diff
@@ -21,6 +22,19 @@ interface CurrentPackProps {
   noRightPadding: boolean;
 }
 
+const TEACHER_FEEDBACK_LINK =
+  'https://docs.google.com/forms/d/e/1FAIpQLSflGeMmY_ff1QllJfpTsWGZdn_xv6dKpPba_evTMwfbvG3FTA/viewform';
+const STUDENT_FEEDBACK_LINK =
+  'https://docs.google.com/forms/d/e/1FAIpQLSeZGNgX4wDvA29stId_Q2toofJN-r12zSP8yBMZ-E9KW5XPWg/viewform';
+
+const useFeedbackLink = () => {
+  const {userType, signInState} = useAppSelector(state => state.currentUser);
+  const isSignedIn = signInState === SignInState.SignedIn;
+  const feedbackLink =
+    userType === 'teacher' ? TEACHER_FEEDBACK_LINK : STUDENT_FEEDBACK_LINK;
```

Comment:
We don't seem to use it universally, but this could probably benefit from a comparison to `UserTypes.TEACHER`, with this import at the top:
```
import {UserTypes} from '@cdo/generated-scripts/sharedConstants';
```